joss_qmc/paper.md

```diff
@@ -0,0 +1,114 @@
+---
+title: 'Quasi-Monte Carlo Methods in Python'
+tags:
+  - Python
+  - SciPy
+  - statistics
+  - Quasi-Monte Carlo methods
+authors:
+  - name: Pamphile T. Roy
+    affiliation: 1
+    corresponding: true
+    orcid: 0000-0001-9816-1416
+  - name: Art B. Owen
+    affiliation: 2
+    orcid: 0000-0001-5860-3945
+  - name: Maximilian Balandat
+    affiliation: 3
+    orcid: 0000-0002-8214-8935
+  - name: Matt Haberland
+    affiliation: 4
+    orcid: 0000-0003-4806-3601
+affiliations:
+    - name: Quansight
+      index: 1
+    - name: Stanford University
+      index: 2
+    - name: Meta
+      index: 3
+    - name: California Polytechnic State University, San Luis Obispo, USA
+      index: 4
+date: 30 August 2022
+bibliography: paper.bib
+
+---
+
+# Summary
+
+
+# Statement of need
+
+NumPy random number generators (`numpy.random`) have become the de-facto
+standard for sampling random numbers in the scientific Python ecosystem.
+These methods are fast and reliable, and the results are repeatable when a
+seed is provided. As a foundational tool, NumPy only provides classical
+Monte Carlo (MC) methods. Sampling in high dimensions with MC produces a lot of
+gaps and clusters of points. When these random numbers are used in algorithms
+(including sampling, numerical integration, optimization)
+to solve deterministic problems, the resulting MC methods have a low
```

Comment:
```suggestion
to solve deterministic problems, the resulting "Monte Carlo" (MC) methods have a low
```
This sentence defines what "Monte Carlo" means for the readers who are unfamiliar with the term, which helps them to understand "Quasi-Monte Carlo" by extension.